src/views/datasources/dataimportcron/actions/DataImportCronActions.tsx

```diff
@@ -24,7 +26,7 @@ const DataImportCronActions: React.FC<DataImportCronActionProps> = ({
 }) => {
   const { t } = useKubevirtTranslation();
   const [isOpen, setIsOpen] = React.useState(false);
```

Comment:
missing type

Reply:
Typescript understands automatically the type from the initial state. false -> boolean. We should specify the type only if we don't initialize the `useState`

Reply:
I think it's a good practice to always specify the type. For code consistency and prevent case of "forgetting" :)